build.sbt

```diff
@@ -16,18 +16,19 @@ lazy val root = (project in file("."))
     licenses += ("Apache-2.0", url("http://www.apache.org/licenses/LICENSE-2.0")),
     scalaVersion := scala212,
     crossScalaVersions := supportedScalaVersions,
-    version      := "3.0-1.1.0",
+    version      := "3.1-1.0.0",
     libraryDependencies ++= Seq(
-      "io.prometheus" % "simpleclient" % "0.9.0",
-      "io.prometheus" % "simpleclient_dropwizard" % "0.9.0",
-      "io.prometheus" % "simpleclient_pushgateway" % "0.9.0",
+      "io.prometheus" % "simpleclient" % "0.11.0",
+      "io.prometheus" % "simpleclient_dropwizard" % "0.11.0",
+      "io.prometheus" % "simpleclient_common" % "0.11.0",
```

Comment:
I have added simpleclient_common to dependencies since it needed for simpleclient_pushgateway and simpleclient. So, we need to include it into fat jar or use `--packages` to add it as well.
See https://github.com/banzaicloud/spark-metrics/pull/75